packages/roosterjs-react-emoji/lib/components/EmojiNavBar.tsx

```diff
@@ -11,24 +11,29 @@ export interface EmojiNavBarProps {
     onClick?: (selected: string) => void;
     currentSelected?: string;
     getTabId?: (selected: string) => string;
+    navbarThemeClassName?: string;
+    navbarButtonThemeClassName?: string,
+    navbarIconThemeClassName?: string,
     strings: Strings;
 }
 
 export default class EmojiNavBar extends React.Component<EmojiNavBarProps, {}> {
     public render() {
-        const { currentSelected, getTabId, strings } = this.props;
+        const { currentSelected, getTabId, strings, navbarThemeClassName = "", navbarButtonThemeClassName = "", navbarIconThemeClassName = "" } = this.props;
```

Comment:
you don't need to give empty as default value, undefined will not be included via css(...) nor className prop